deploy/charts/cert-manager/templates/webhook-deployment.yaml

```
@@ -146,10 +146,15 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-          {{- if .Values.webhook.config }}
+          {{- if or .Values.webhook.config (gt (len .Values.webhook.volumeMounts) 0) }}
```

I think we can simplify this check, please correct me if I'm wrong:
```suggestion
          {{- if or .Values.webhook.config .Values.webhook.volumeMounts }}
```